Src/BootCamp.Chapter/BalanceParser.cs

```diff
@@ -0,0 +1,167 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class BalanceParser
+    {
+        public static BalanceStats FindHighestBalance(string[] balances)
+        {
+            CultureInfo.CurrentCulture = new CultureInfo("en-GB");
+            var persons = new StringBuilder();
+            var personWithHighestBalance = new Person("", decimal.MinValue);
+
+            for (int i = 0; i <= balances.Length - 1; i++)
+            {
+                var currentPersonData = balances[i].Split(',');
+
+                var highestAmountOfPerson = decimal.Parse(currentPersonData[1..].Max(), NumberStyles.Currency);
+
+                if (highestAmountOfPerson > personWithHighestBalance.GetAmount())
+                {
+                    personWithHighestBalance = new Person(currentPersonData[0], highestAmountOfPerson);
+                    persons.Clear();
+                }
+
+                if (highestAmountOfPerson >= personWithHighestBalance.GetAmount())
+                {
+                    AddPerson(currentPersonData[0], persons);
+                }
+            }
+
+            if (persons.ToString().Contains(", "))
+            {
+                ReplaceCommaWithAnd(persons);
+            }
+
+            return new BalanceStats(persons, personWithHighestBalance.GetAmount());
+        }
+
+        private static void AddPerson(string person, StringBuilder sb)
+        {
+            if (sb.Length == 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)